deployment/aws-beanstalk.md

```diff
@@ -0,0 +1,6 @@
+# Deployment in AWS Elastic Beanstalk
+
+For instructions on managing the Arclight and Blacklight demo sites see:
+
+ https://docs.google.com/document/d/1EjPJf5-__7ca17xjnNOOmSUY9sGjnRMOiRalOM-M2Z4/edit
```

Comment:
Is it desirable that the linked Google Doc isn't viewable when I'm logged into to Google via my Stanford account?